### ISSUE TITLE
Update?

### ISSUE THREAD

**Issue by shadowking1443** [PunishedPineapple/ItemUse#1]
Hi,

Just wanted to see if this plugin would be getting an update post DT.

**Comment by PunishedPineapple** [comment]
Hi, so there is a working version of the plugin available (at least it was working shortly after 7.11), but it has one issue: when co-installed with Allagan Tools, and when you have item tooltips enabled for Allagan Tools and coffer job icons enabled for this plugin, they sometimes swap order in the tooltip (item quantities first then coffer jobs, or vise versa).  This is because Allagan Tools changed how it appends text to the tooltip in 7.1, and I was unable to easily figure out exactly why the changes he made do what they do.  I kinda can't even handle daily life anymore at this point, so I have no idea when I will get it together enough to dig into that.  I am sorry for being so unreliable.

If you really want it and don't mind installing the testing release of the plugin, you can (iirc) right click on the plugin in the plugin installer and select the testing version.  Normally I would not recommend doing that in general if you're not planning to actually do testing, but in this case, the only difference from what would be the final product is the layout inconsistency when paired with Allagan Tools.  It is otherwise the same as it was before 7.1 (besides the Dalamud API changes for that patch).  I was tempted to just release it to stable as-is, but people tend to not read patch notes, so I don't want the plugin feedback channel to get flooded with people reporting a known issue that may not be fixed for months since that would just annoy the other devs.

**tl;dr:** The released testing version of the plugin should be functional, but I don't know when I will have it in me to fix the minor visual bug that is keeping it from releasing to stable.

**Comment by PunishedPineapple** [comment]
It is back on stable now, although it still has the formatting issue.
